CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 All notable changes to this project will be documented in this file.
 
+## [4.4.2] - 2025-07-23
+
+### Security
+
+- Update dependencies
+
+### Fixed
+
+- Don't require JSDoc params & return in TS (#35426 by @diondiondion)
+- Disable `useDrag` hook when main menu is not openable (#35414 by @diondiondion)
+- Add `lang` attribute to current composer language in alt text modal (#35412 by @diondiondion)
+- Fix quote posts styling on notifications page (#35411 by @diondiondion)
+- Improve a11y of custom select menus in notifications settings (#35403 by @diondiondion)
+- Fix selected item in poll select menus is unreadable in Firefox (#35402 by @diondiondion)
+- Only remove pointer-events when necessary (#35390 by @diondiondion)
+- Update age limit wording (#35387 by @diondiondion)
+- Fix support for quote verification in implicit status updates (#35384 by @ClearlyClaire)
+- Always give local quote of remote posts a quote request URI (#35383 by @ClearlyClaire)
+- Refactor `ActivityPub::Activity::Accept` and `ActivityPub::Activity::Reject` specs (#35382 by @ClearlyClaire)
+- Fix `quoteAuthorization` type in JSON-LD context (#35380 by @ClearlyClaire)
+- Improve `Dropdown` component accessibility (#35373 by @diondiondion)
+- Fix quote attributes missing from Mastodon's context (#35354 by @ClearlyClaire)
```

Comment:
I would merge two of these changes:
```suggestion
- Improve `Dropdown` component accessibility (#35373 by @diondiondion)
- Fix processing some incoming quotes failing because of missing JSON-LD context (#35354 and #35380 by @ClearlyClaire)
```